Sources/SectionScrubber.swift

```diff
@@ -224,7 +224,11 @@ public class SectionScrubber: UIView {
             return indexPath
         }
         for indexPath in collectionView.indexPathsForVisibleSupplementaryElementsOfKind(UICollectionElementKindSectionHeader) {
-            let view = collectionView.supplementaryViewForElementKind(UICollectionElementKindSectionHeader, atIndexPath: indexPath)
+            #if swift(>=2.3)
+                let view = collectionView.supplementaryViewForElementKind(UICollectionElementKindSectionHeader, atIndexPath: indexPath)!
```

Comment:
well if it will be an optional by default in the future we'll need to deal with the optional even more right?